code/lib/theming/src/themes/light.ts

```diff
@@ -6,7 +6,7 @@ const theme: ThemeVars = {
 
   // Storybook-specific color palette
   colorPrimary: '#FF4785', // coral
-  colorSecondary: '#1EA7FD', // ocean
+  colorSecondary: '#029CFD', // ocean
```

Comment:
seems like these are shared between themes - could be exported as constants in base and used in the light and dark files

Reply:
Great! On it!

Reply:
I ran into a hiccup. I implemented it as you suggested, but `convert.js` gets a bit mad unless I add them there. I am not sure the best way to do that.